maintainer/maintainer.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import ts3
+
+import configuration
+from channels import ChannelsMaintanance
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+
+def establish_connection():
```

Comment:
Is this function needed?

Reply:
not needed, we are using API at this moment